src/components/Anchor/index.js

```diff
@@ -21,13 +21,14 @@ const prefixByType = {
 const Anchor = ({
   url,
   children,
+  className = '',
   isSeparateTab = false,
   type = 'inline',
 }) => (
   <a
     href={url}
     target={isSeparateTab ? '_blank' : '_self'}
-    className={classNameByType[type]}
+    className={`${classNameByType[type]} ${className}`}
```

Comment:
```
<Anchor className='f6 link dim br1 ba ph3 pv2 mb2 dib black' />
Now anchor looks like a Button, for which there is already a Button component.
```

Is that abuse? I mean, at the end of the day, if I want my anchor to look like a button or whatever it should behave as I expected and not have the iron claw className clamp down. 

I could also just skip it and use an `<a>` yolo style, but trying to go with the program.